### ISSUE TITLE
Possible to use track_kalman

### ISSUE THREAD

**Issue by lehador1** [AlexeyAB/darknet#2948]
I have list of image files. Is it possible to use track_kalman on this and show it?

**Comment by rveck** [comment]
Yes, try this project, https://github.com/Qidian213/deep_sort_yolov3
